File: app/src/main/java/ardjomand/leonardo/arcmovies/model/UpcomingMovies.java
package ardjomand.leonardo.arcmovies.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by unity on 25/02/18.
 */

public class UpcomingMovies {

    @SerializedName("results")
    @Expose
    private List<UpcomingMovie> results = null;
    @SerializedName("page")
    @Expose
    private Integer page;
    @SerializedName("total_results")
    @Expose
    private Integer totalResults;
    @SerializedName("total_pages")
    @Expose
    private Integer totalPages;

    public List<UpcomingMovie> getResults() {
        return results;
    }

    public void setResults(List<UpcomingMovie> results) {
        this.results = results;
    }

    public Integer getPage() {
        return page;
    }

    public void setPage(Integer page) {
        this.page = page;
    }

    public Integer getTotalResults() {
        return totalResults;
    }

    public void setTotalResults(Integer totalResults) {
        this.totalResults = totalResults;
    }

    public Integer getTotalPages() {
        return totalPages;
    }

    public void setTotalPages(Integer totalPages) {
        this.totalPages = totalPages;
    }

    @Override
    public String toString() {
        return "UpcomingMovies{" +
                "results=" + results +
                ", page=" + page +
                '}';
    }
}

File: app/src/main/java/ardjomand/leonardo/arcmovies/model/UpcomingMovies.kt
package ardjomand.leonardo.arcmovies.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by unity on 25/02/18.
 */

data class UpcomingMovies(
        @SerializedName("results")
        @Expose
        var results: List<UpcomingMovie>? = null,

        @SerializedName("page")
        @Expose
        var page: Int? = null,

        @SerializedName("total_results")
        @Expose
        var totalResults: Int? = null,

        @SerializedName("total_pages")
        @Expose
        var totalPages: Int? = null) {

    override fun toString(): String {
        return "UpcomingMovies{" +
                "results=" + results +
                ", page=" + page +
                '}'.toString()
    }
}